### PR TITLE
Suit Storage Resprite by Retlaw34

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

A port of https://github.com/shiptest-ss13/Shiptest/pull/500
resprites the standard suit storage, adds an industrial subtype with a different sprite (from eris)
image

## Why It's Good For The Game

the current one is dogshit and uses almost 0 shared colors and has random transparent parts

## Changelog

:cl: Retlaw34
imageadd: New suit storage sprites
/:cl:

### CODE REVIEW

File: code/game/machinery/suit_storage_unit.dm [hunk 1]
```diff
@@ -51,10 +52,6 @@
 	/// How fast it charges cells in a suit
 	var/charge_rate = 250
 
-/obj/machinery/suit_storage_unit/Initialize(mapload)
-	. = ..()
-	interaction_flags_machine |= INTERACT_MACHINE_OFFLINE
```
Comment (Fikou) [inline code/game/machinery/suit_storage_unit.dm:56]: this was added at some point. suit storage units have behavior to be opened up with a crowbar when unpowered so it makes no sense

**Comment by TheBonded** [comment]
Can we get a photo of the new sprites?

**Review by Imaginos16 — APPROVED** [review]
Quality sprites!

**Review by Imaginos16 — CHANGES_REQUESTED** [review]
Actually, while it is looking good, there's a tiny oopsie with a white pixel showing up on the top left corner of the classic_super sprite

image

**Review by Imaginos16 — APPROVED** [review]
As I was informed, this was just icondiffbot having a stroke. Incredible. Approved, again.

**Pull request merged**